### PR TITLE
Fix-bugs-for-demo

### PR DESCRIPTION
**Pull request by darmhoo**
Fix pagination, stopped mutating props, stopped autoscrolling

### CODE REVIEW

File: src/views/admin/Dashboard.vue [hunk 1]
```diff
@@ -1,1262 +0,0 @@
-<template>
```
Comment (teebarg) [inline src/views/admin/Dashboard.vue:1]: Any reason you're deleting this file?

**Comment by teebarg** [comment]
Can you add details what the PR is doing?

**Comment by darmhoo** [comment]
> Any reason you're deleting this file? 

Linting asked me to rename, No single word component

**Comment by teebarg** [comment]
Screenshot 2023-09-21 at 12 50 37

This is still broken.

**Review by teebarg — APPROVED** [review]

**Pull request merged**